src/status_im2/contexts/onboarding/create_password/view.cljs

```diff
@@ -189,11 +190,18 @@
   (let [scroll-view-ref  (atom nil)
         scroll-to-end-fn #(js/setTimeout ^js/Function (.-scrollToEnd @scroll-view-ref) 250)]
     (fn []
-      [:<>
-       [background/view true]
-       [rn/scroll-view
-        {:ref                     #(reset! scroll-view-ref %)
-         :style                   style/overlay
-         :content-container-style style/content-style}
-        [navigation-bar/navigation-bar {:on-press-info #(js/alert "Info pressed")}]
-        [password-form {:scroll-to-end-fn scroll-to-end-fn}]]])))
+      [safe-area/consumer
```

Comment:
Hi @alwx 

I suggest not to create a reagent component on the fly in an inline function:

```clojure
[safe-area/consumer
  (fn [props] ;; <- this component
   [ ;; view code
    ])]
```

This could lead us to a performance downgrade and problems while using hooks (https://github.com/status-im/status-mobile/issues/15552#issue-1649290578), and I would also suggest to use the pattern I commented there.

Another option, probably more clean (I'm not sure yet) would be to use the `useSafeAreaInsets` hook since I think you only need that `top` value, so probly using a hook requires less refactoring.